### PR TITLE
bootstrap + navbar and models + migrations + associations

### PR DESCRIPTION
**Pull request by sunniraleigh**
- add navbar and boostrap and `app/views/layouts/application.html.erb`
- setup models + migrations + associations

### CODE REVIEW

File: app/assets/stylesheets/logins.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/*
```
Comment (kvangreen) [inline app/assets/stylesheets/logins.css:1]: Ideally, we'll only need one stylesheet for the whole app—the theme/styles should be the same across pages, and the only differences will be sections that we can determine with classes or ids.

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<img src="/assets/download.jpg" alt = "Background">
```
Comment (epartakki) [inline app/views/home/index.html.erb:1]: seems irrelevant!

Comment (sunniraleigh) [inline app/views/home/index.html.erb:1]: this file is irrelevant now

**Review by epartakki — APPROVED** [review]
Great work Sunni!

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -9,7 +9,6 @@ def create
     @user = User.find_by(username: params[:username])
     if @user && @user.authenticate(params[:password])
 
-
         session[:user_id] = @user.id
 
         redirect_to '/welcome'
```
Comment (kvangreen) [inline app/controllers/sessions_controller.rb:14]: It might make more sense to change welcome.html.erb to index.html.erb

**Review by luciebrock — APPROVED** [review]
nice job lad

File: app/views/home/index.html.erb (hunk 2 above)
Comment (kvangreen) [inline app/views/home/index.html.erb:1]: What is this? Why is it here?

File: test/controllers/home_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (kvangreen) [inline test/controllers/home_controller_test.rb:1]: What is this for?

**Review by kvangreen — APPROVED** [review]
Overall good

**Review by amarrap — APPROVED** [review]
good stuff

File: app/controllers/sessions_controller.rb (hunk 3 above)
Comment (sunniraleigh) [reply app/controllers/sessions_controller.rb:14]: good point

File: test/controllers/home_controller_test.rb (hunk 4 above)
Comment (sunniraleigh) [reply test/controllers/home_controller_test.rb:1]: tests auto-generate because of a gem. don't worry about it rn!
